_posts/2017-03-20-lightning-q1-2017.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Lightning Talks - Three Useful Additions to Any Developer Toolbox in Q1 2017
+date: 2017-03-20
+author: Ro Wang
+tags: skillshare efficiency docker
+featured_image: "/blog/lightning-17q1-hero.jpg"
+featured_image_alt: "Q1 2017 Lightning Talks"
+featured_image_height: "917px"
+featured_image_width: "1474px"
+summary: What we've added to our developer toolboxes in Q1 2017!
+---
+
+_This post is part of a new series called "Lightning Talks."_
+
+The Web Development world evolves quickly, and keeping abreast of new tools and methodologies while simultaneously delivering client work is tough.
```

Comment:
consider lowercasing "Web Development"? not sure if it should be capitalized, I usually don't.